app/controllers/partners/requests_controller.rb

```diff
@@ -43,14 +43,16 @@ def create
     end
 
     def validate
-      @partner_request = Partners::RequestCreateService.new(
+      create_service = Partners::RequestCreateService.new(
         request_type: "quantity",
         partner_user_id: current_user.id,
         comments: partner_request_params[:comments],
         item_requests_attributes: partner_request_params[:item_requests_attributes]&.values || []
-      ).initialize_only
+      )
 
-      if @partner_request.valid?
+      create_service.initialize_only
```

Comment:
Just curious why this one follows a different pattern than the others? Seems like `initialize_only` returns the service anyway?

Reply:
Because I did this one first and was trying to make it _more like_ the `create_service.call` in the `def create` above. I'll make the others like this one, and then if you don't like it then I'll make all of them the original way :)

Reply:
There we go -- now they all look exactly the same as a dedicated action on the `create_service` instance. The code after that uses the instance, which whether it is the `validate` or the `create` can always be treated as a service wrapper around a request.

Reply:
er... hm. ok one min...

Reply:
@dorner I didn't realize that the family and children request services were yet another wrapper layer for the request service. Kinda weird. But rather than fix it I went back the other way and now they all look the same in the controllers.